### PR TITLE
Add CSKY support

### PR DESCRIPTION
**Pull request by Dirreke**
Add support for arch CSKY

### CODE REVIEW

File: Makefile.system [hunk 1]
```diff
@@ -873,6 +873,16 @@ endif
 endif
 endif
 
+ifeq ($(ARCH), csky)
+NO_BINARY_MODE	= 1
+BINARY_DEFINED	= 1
+endif
+
+ifeq ($(ARCH), ck860v)
```
Comment (martin-frbg) [inline Makefile.system:881]: the ifeq (ARCH,ck860v) part looks a bit redundant here, assuming csky is going to be the ARCH and ck860v the TARGET (or eventually, one of several cpu targets under the csky architecture) ?

Comment (Dirreke) [reply Makefile.system:881]: Thanks. My bad, I will delete it.

**Comment by martin-frbg** [comment]
Thank you - can you please also update TargetList.txt ? If I'm reading your PR correctly, you are adding CK860FV and a CSKY where I assume the latter will remain a "generic" target which will keep using the C kernels while CK860FV may get optimized code later ?

**Comment by Dirreke** [comment]
Thanks, I pull the wrong branch, I will create a new PR to solve it. #4438

**Pull request closed without merging**